lib/readline.js

```diff
@@ -379,7 +379,10 @@ Interface.prototype._tabComplete = function() {
         var width = completions.reduce(function(a, b) {
           return a.length > b.length ? a : b;
         }).length + 2;  // 2 space padding
-        var maxColumns = Math.floor(self.columns / width) || 1;
+        var maxColumns = Math.floor(self.columns / width);
```

Comment:
Leaving out the ` || 1`, i.e. allowing `maxColumns==0` (e.g. set `self.columns==1` while `width>1`) is probably not a good idea, because it will send handleGroup into an infinite loop (via `minRows==Infinity`)...